model/common/src/icon4py/model/common/grid/geometry.py

```diff
@@ -571,6 +590,7 @@ def create_auxiliary_coordinate_arrays_for_orientation(
     cell_lon: fa.CellField[ta.wpfloat],
     edge_lat: fa.EdgeField[ta.wpfloat],
     edge_lon: fa.EdgeField[ta.wpfloat],
+    backend: Optional[gtx_backend.Backend],
```

Comment:
```suggestion
    backend: Optional[gtx_backend.Backend] = None,
```
not sure if you need this

Reply:
I don't really want to set a default, the user should provide it.